test/gpu/native/dataPingPong.compopts

```diff
@@ -0,0 +1 @@
+-sdebugBulkTransfer
```

Comment:
I'd like to understand what this flag is doing a little more. I'm guessing it's whats producing this message:

`In DefaultRectangular._simpleTransfer(): Alo=(1), Blo=(1), len=10, elemSize=8`

So this is for the bulk transfer optimization right. So if I did something like

```
A[0..5] = B[0..5];
A[6..n] = B[6..n];
```

would it show me a single message that's aggregating the transfer?

Reply:
The prediff of this filter grabs only this line. It is actually more verbose than this single line.

And yes, for those two lines, I expect this flag to generate some output.

As a side node `-s*` flags are config params. You can look for `debugBulkTransfer` in the module code to see what exactly it enables. Bulk transfer is completely module-driven.